docs/dashboard/visualization-date-and-time-controls.mdx

```diff
@@ -1,11 +1,15 @@
 ---
-title: "Select timeframes to visualize"
-description: "Netdata's dashboard features a rich timeframe selector, with useful defaults and rich customization, to help you narrow your focus when troubleshooting issues or anomalies."
+title: "Visualization date and time controls"
+description: "Netdata's dashboard features powerful date visualization controls that include a time control (play, pause, force play), a timezone selector and a rich date and timeframe selector, with useful defaults and rich customization, to help you narrow your focus when troubleshooting issues or anomalies."
 type: how-to
-custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/dashboard/select-timeframes.mdx
+custom_edit_url: https://github.com/netdata/netdata/edit/master/docs/dashboard/visualization-date-and-time-controls.mdx
 ---
 
-# Pick timeframes to visualize
+# Visualization date and time controls
+
+## Date and time selector
+
+### Pick timeframes to visualize
 
 While [panning through time and zooming in/out](/docs/dashboard/interact-charts.mdx) from charts is helpful when
 you're looking a recent history, or want to do granular troubleshooting, what if you want to see metrics from 6 hours
```

Comment:
```suggestion
for looking at recent history, or for granular troubleshooting, if you want to see metrics for the previous 6 hours
```

Reply:
it was missing a when, don't now how it gotten taken out, so it should read:

"While panning through time and zooming in/out from charts is helpful when you're looking a recent history..." I think that was why you suggested the ".. for looking"